lib/screens/details/components/add_to_cart.dart

```diff
@@ -25,13 +26,17 @@ class AddToCart extends StatelessWidget {
             decoration: BoxDecoration(
               borderRadius: BorderRadius.circular(18),
               border: Border.all(
-                color: MediaQuery.of(context).highContrast ? Theme.of(context).primaryColorLight : product.color,
+                color: MediaQuery.of(context).highContrast
```

Comment:
Shouldn't this use the new `HighContrast.isHighContrastActive()`?

Reply:
Same for all following files

Reply:
Nope, essa classe é uma ideia em construção para a resolução do bug da necessidade de um botão em Android